docs/data/date-pickers/custom-field/JoyV6Field.tsx

```diff
@@ -121,15 +98,6 @@ const JoyDatePicker = React.forwardRef(
         ref={ref}
         {...props}
         slots={{ ...props.slots, field: JoyDateField }}
-        slotProps={{
-          ...props.slotProps,
-          field: {
-            ...props.slotProps?.field,
-            formControlSx: {
-              flexDirection: 'row',
```

Comment:
No idea why we passed this weird `sx`.
It was totally breaking the layout if a label was provided